activestorage/lib/active_storage/attached/model.rb

```diff
@@ -135,18 +152,18 @@ def #{name}
             @active_storage_attached[:#{name}] ||= ActiveStorage::Attached::Many.new("#{name}", self)
           end
 
-          def #{name}=(attachables)
+          define_method("#{name}=") do |attachables|
             if ActiveStorage.replace_on_assign_to_many
               attachment_changes["#{name}"] =
```

Comment:
If you rework below lines, also rework this one? Should you replace with "to_s"? I guess there is a reason why it is interpolated in a string (?)

Reply:
i will revert this change